war/src/main/less/pluginSetupWizard.less

```diff
@@ -492,7 +495,6 @@
 								background: @badge-color;
 								color: #fff;
 							}
-							outline: none;
```

Comment:
@fqueiruga ``outline:none`` was removed, because it was duplicated in the block.